src/_TextInputInnerAction.tsx

```diff
@@ -5,18 +5,38 @@ import {ButtonProps} from './Button'
 import {BetterSystemStyleObject, merge, SxProp} from './sx'
 
 type TextInputActionProps = Omit<React.HTMLProps<HTMLButtonElement>, 'aria-label' | 'size'> & {
+  /** @deprecated Text input action buttons should only use icon buttons */
+  children?: React.ReactNode
   /** Text that appears in a tooltip. If an icon is passed, this is also used as the label used by assistive technologies. */
   ['aria-label']?: string
   /** The icon to render inside the button */
   icon?: React.FunctionComponent<IconProps>
   /**
+   * @deprecated Text input action buttons should only use the 'invisible' button variant
```

Comment:
Does this make it a breaking change? Would there be new warnings when someone upgrades primer/react?

How do we usually deprecate props?

_Side note: Are there any users of `TextInput.Action`? Can we get away with this in patch_

Reply:
> Does this make it a breaking change?

No, if they're still using `variant`, it will still work

---

> Would there be new warnings when someone upgrades primer/react?

I _think_ it would just be in their IDE

---

> How do we usually deprecate props?

I don't think we have a defined process yet. Maybe @colebemis knows

---

> Side note: Are there any users of TextInput.Action? Can we get away with this in patch

I don't think so. Even if there were, I think we can get away with this in a patch.